packages/mirinae/src/inputs/dropdown/filterable-dropdown/PFilterableDropdown.vue

```diff
@@ -152,105 +186,95 @@ const handleClearSelection = () => {
     emit('clear-selection');
 };
 const handleUpdateSearchText = debounce(async (searchText: string) => {
-    proxySearchText.value = searchText;
+    state.proxySearchText = searchText;
     if (!props.disableHandler) {
         await reloadMenu();
     }
 }, 200);
-watch([() => props.menu, proxyVisibleMenu], async ([, visibleMenu]) => {
-    if (visibleMenu && !props.disableHandler) {
-        await initiateMenu();
-    }
-}, { immediate: true });
-
-/* deletion */
-const showDeleteAllButton = computed(() => {
-    if (props.disabled) return false;
-    if (props.readonly) return false;
-    return !!proxySelected.value.length;
-});
 const handleClickDeleteAll = () => {
-    if (proxySelected.value.length) {
-        updateSelected([]);
+    if (state.proxySelectedItem.length) {
+        handleUpdateSelected([]);
     }
 };
-const handleTagDelete = (item: FilterableDropdownMenuItem, idx: number) => {
-    const selectedClone = [...proxySelected.value];
+const handleTagDelete = (item: DropdownMenuItem, idx: number) => {
+    const selectedClone = [...state.proxySelectedItem];
     selectedClone.splice(idx, 1);
-    updateSelected(selectedClone);
+    handleUpdateSelected(selectedClone);
```

Comment:
suggestion: We discussed it and decided not to call event handler but just bind it at event.
I think previous name is better in this case.